store/user_inmemory.go

```diff
@@ -15,19 +16,22 @@ type UserInMemory struct {
 }
 
 func NewUserInMemory() *UserInMemory {
+	pass1, _ := bcrypt.GenerateFromPassword([]byte("kubernetes"), bcrypt.DefaultCost)
+	pass2, _ := bcrypt.GenerateFromPassword([]byte("golang"), bcrypt.DefaultCost)
+
 	return &UserInMemory{
 		users: []gitpod.User{{
 			ID:       "25558000-2565-48dc-84eb-18754da2b0a2",
 			Username: "metalmatze",
 			Name:     "Matthias Loibl",
 			Email:    "metalmatze@example.com",
-			Password: "encrypted with bcrypt",
+			Password: string(pass1),
```

Comment:
`string([]non-UTF8-bytes)` will ick out everywhere, `hex.EncodeToString(pass1)` is probably your friend if you insist on using passwords